searx/engines/hex.py

```diff
@@ -46,10 +46,9 @@ def response(resp):
                 "content": meta.get("description", ""),
                 "version": meta.get("latest_version"),
                 "maintainer": ", ".join(meta.get("maintainers", [])),
-                "publishedDate": publishedDate,
-                "tags": tags,
-                "homepage": meta.get("links", {}).get("homepage"),
-                "source_code_url": meta.get("links", {}).get("github"),
+                "published_date": published_date,
```

Comment:
Please don't change that, it won't continue to work that way. The field in the html template still uses `publishedDate` (as any template in SearXNG does).

I agree that `published_date` would be better, but that should be done in a different PR and also update the documentation.